docs/configuration.asciidoc

```diff
@@ -5,28 +5,35 @@ Please only make changes in configuration.asciidoc.ftl
 ////
 [[configuration]]
 == Configuration
-To adapt the Elastic APM agent to your needs,
-you can configure it using different configuration sources,
-which have different naming conventions for the property key.
-In descending order of precedence:
+The Elastic APM agent can be configured through different configuration sources,
+with the following descending order of precedence:
 
 [arabic]
 . {apm-app-ref}/agent-configuration.html[Central configuration]
+. `elasticapm.properties` file, by default in the same directory the agent jar. +
```

Comment:
I think we could do a better job of flagging which configuration sources allow dynamic changes. Maybe we could use asciidoc footnotes for that.

```suggestion
. {apm-app-ref}/agent-configuration.html[Central configuration] footnote:dynamic[Configuration options marked with Dynamic true can be changed at runtime with this configuration source]
. `elasticapm.properties` file footnote:dynamic[], by default in the same directory the agent jar. +
```

disclaimer: I've never used footnotes before and have not tested this renders properly